phold/Node.cpp

```diff
@@ -194,6 +194,7 @@ void Node::serialize_order(SST::Core::Serialization::serializer& ser) {
     SST_SER(smallPayload);
     SST_SER(largePayload);
     SST_SER(largeEventFraction);
+    SST_SER(additionalData);
```

Comment:
Looks like you do this in <https://github.com/hpc-ai-adv-dev/sst-benchmarks/pull/36>. Not sure if you included it in this PR intentionally or not.